be2-scala/build.sbt

```diff
@@ -76,7 +76,7 @@ copyProtocolTask := {
 }
 
 // Add the copyProtocolTask to compile and test scopes
-(Compile/ compile) := ((Compile/ compile) dependsOn copyProtocolTask).value
+Compile / compile := (Compile / compile).dependsOn(copyProtocolTask).value
```

Comment:
The syntax of the line changed a bit, and it is not the same as the line 80. Was it intentional? If not, maybe just sticking with the previous one would be better in terms of code uniformity. (minor comment btw)

Reply:
You are right! Added consistency throughout the whole file =)